src/modules/components/vote/voteDataListItem/voteDataListItemStructure/voteDataListItemStructure.tsx

```diff
@@ -0,0 +1,76 @@
+import classNames from 'classnames';
+import {
+    DataList,
+    NumberFormat,
+    Tag,
+    formatterUtils,
+    type IDataListItemProps,
+    type TagVariant,
+} from '../../../../../core';
+import { type ICompositeAddress } from '../../../../types';
+import { addressUtils } from '../../../../utils';
+import { MemberAvatar } from '../../../member';
+
+export interface IVoteDataListItemStructureProps extends IDataListItemProps {
+    /**
+     * The account details of the voter.
+     */
+    voter: ICompositeAddress;
+    /**
+     * The way the voter used their voting power.
+     */
+    voteIndicator: 'yes' | 'no' | 'abstain' | 'approved' | 'none';
+    /**
+     * If token-based voting, the amount of token voting power used.
+     */
+    voteTokenAmount?: number | string;
+    /**
+     * If token-based voting, the symbol of the token voted.
+     */
+    voteTokenSymbol?: string;
```

Comment:
We don't need to add the `vote` prefix as the component name already defines the vote context, you can just call this `tokenSymbol`